testing/tests/index.test.ts

```diff
@@ -1513,6 +1523,7 @@ test("home page should have a /index.json file with pullRequestsData", () => {
   const builtFolder = path.join(buildRoot, "en-us");
 
   const jsonFile = path.join(builtFolder, "index.json");
+  // @ts-ignore
   const { hyData: { recentContributions } = {} } = JSON.parse(
```

Comment:
Don't think this `ts-ignore` is necessary if you do:
`const { hyData: { recentContributions = {} } = {} } = JSON.parse(`